### PR TITLE
Feature/atr 177 merge after Feature/atr 219

### PR DESCRIPTION
**Pull request by saloponov**

### CODE REVIEW

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 1]
```diff
@@ -342,4 +342,16 @@
...
     <value>A PXGraph instance should not start PXLongOperation during the initialization phase</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
+  </data>
+  <data name="PX1055Fix2" xml:space="preserve">
+    <value>Alter the definition of the field bound to identity column to make it a non-key field and leave all other key fields untouched</value>
+  </data>
+  <data name="PX1055Fix3" xml:space="preserve">
+    <value>Remove the definition of the field bound to identity column and leave all other key fields untouched (it will be also very helpful to show a hint suggesting to drop the identity column, to which this key field is bound, from database)</value>
```
Comment (saloponov) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: @dropsonic , how I can do hint suggestion?

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 2]
```diff
@@ -342,4 +342,16 @@
...
+  </data>
+  <data name="PX1055Title" xml:space="preserve">
+    <value>Invalid primary key of the DAC. Use a single identity field or one or multiple natural fields for the key.</value>
```
Comment (saloponov) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: @k-s-popova , please verify string

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 3]
```diff
@@ -342,4 +342,16 @@
   <data name="PX1054Title" xml:space="preserve">
     <value>A PXGraph instance should not start PXLongOperation during the initialization phase</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
```
Comment (saloponov) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: @k-s-popova , please verify string

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 4]
```diff
@@ -342,4 +342,16 @@
...
+  </data>
+  <data name="PX1055Fix2" xml:space="preserve">
+    <value>Alter the definition of the field bound to identity column to make it a non-key field and leave all other key fields untouched</value>
```
Comment (saloponov) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: @k-s-popova , please verify string

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 3 above)
Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @saloponov , consider to change as follows:
Use the field bound to the identity column as the only key field in the DAC

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 4 above)
Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @saloponov , please change as follows:
Make the field bound to the identity column be a non-key field and leave all other key fields untouched

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 2 above)
Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @saloponov , this one has already been verified

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 1 above)
Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @saloponov , consider the following (if the hind is displayed along with the fix message):
Remove the definition of the field bound to the identity column and leave all other key fields untouched. (You should drop the corresponding identity column from the database manually.)

**Pull request closed without merging**